theme/templates/includes/access-row.html

```diff
@@ -1,133 +1,89 @@
 {% load hydroshare_tags %}
 
-{% for u in entries %}
-<tr id="row-id-{{ u.pk }}"
-    {% if access != "Is owner" %}
-    {% elif entries|length == 1 or not is_owner_user or u.pk == quota_holder.pk %}
-    class="hide-actions"
-    {% endif %}>
+<tr v-for="(user, index) in this.users"
+    :class="{ 'hide-actions': selfAccessLevel !== 'owner' && (user.access == 'owner' && hasOnlyOneOwner || user.id === quotaHolder.id), loading: user.loading}">
     <td>
         <table class="user-scope">
-            {% if access_type == "user" %}
-            <tr>
+            <tr v-if="user.user_type === 'user'">
                 <td>
-                    {% if u.userprofile.picture %}
-                    <div style="background-image: url('{{ u.userprofile.picture.url }}');"
-                         class="profile-pic-thumbnail round-image"></div>
-                    {% else %}
-                    <div class="profile-pic-thumbnail round-image user-icon"></div>
-                    {% endif %}
+                    <div v-if="user.pictureUrl"
+                         :style="{backgroundImage: 'url(' + user.pictureUrl + ')'}"
+                         class="profile-pic-thumbnail round-image">
+                    </div>
+
+                    <div v-else class="profile-pic-thumbnail round-image user-icon"></div>
                 </td>
                 <td>
-                    <div data-col="name" class="user-name">
-                        <a href="/user/{{ u.id }}">{{ u|best_name }}</a>
+                    <div class="user-name">
+                        <a :href="'/user/' + user.id">${user.best_name}</a>
                     </div>
-                    <div data-col="user-name"
-                          class="user-username-content">{{ u.username }}</div>
+                    <div class="user-username-content">${user.user_name}</div>
                     <div>
```

Comment:
Ok I think you set up a custom handler for $... is another valid (but less readable) approach to use the django tags?

{% templatetag openvariable %} user.user_name {% templatetag closevariable %} 

when jQuery is present, I try to avoid hijacking the $ symbol...

Reply:
I avoided using django escape tags because we might want to keep django tags running to render specific content inside Vue templates.

Since this runs in the HTML and jQuery runs on the scripts, I don't think there will ever be a conflict.

However, we could define better delimiters so I'm open to suggestions. One that comes to mind is double brackets: `[[ some_expression ]]`

Reply:
In that case, the way you have originally designed it is probably best.